packages/webview_flutter/webview_flutter_platform_interface/pubspec.yaml

```diff
@@ -14,7 +14,7 @@ dependencies:
   flutter:
     sdk: flutter
   meta: ^1.7.0
-  plugin_platform_interface: ^2.1.0
+  plugin_platform_interface: ^2.1.6
```

Comment:
Why is this changing? I don't think anything changed in that version that we could be depending on.